dashboards/system_tables/docs/job_comparison.md

```diff
@@ -0,0 +1,57 @@
+# Job Comparison Report
+The [Job Comparison Report](https://datastudio.google.com/c/u/0/reporting/c30ea9f1-31d4-401a-8ab7-b3f869b75d29/page/jsQ0B/) gives an overview of how to compare job performance of two jobs, given two job IDs. 
+This is meant to allow for a side-by-side troubleshooting for understanding why one query may have performed much slower than other.
+
+The report allows you to compare the jobs' overall performance as well as the state of BigQuery for your organization while each job ran.
+
+All timestamps in this report are in UTC.
+
+This report leverages the [job comparison statistics query](../sql/job_comparison_statistics.sql), [slow concurrency query](../sql/job_concurrency_comparison_slow.sql), 
+[fast concurrency query](../sql/job_concurrency_comparison_fast.sql), [slow job analyzer query](../sql/job_analyzer_slow.sql), and [fast job analyzer query](../sql/job_analyzer_fast.sql).
+
+### Job ID Inputs
+This report allows users to enter in two job IDs: one from a fast job and one from a slow job. Please note that these should be similar jobs, such as scheduled job run at different timeframes or jobs that read in different partitions from the same underlying source table.
+When a user adds an ID, all charts and scorecards on the page will be updated to use those filters. 
+
+Below is an example of where to input your job IDs.
+
+You will need to input both as they are configured as query parameters, `@job_param` and `@job_param_2`, in the underlying SQL.
```

Comment:
I did job_param and _2 so that it's easy when copying to just add a _2 vs. changing the weird slow vs. fast may not be as clear. But happy to change if you think it's a blocker.